app.rb

```diff
@@ -15,10 +15,12 @@
 end
 
 get '/' do
-  { 'message': 'Hello world!' }
+  { 'message': 'Ola!' }
```

Comment:
Why did you change this?

```suggestion
  { 'message': 'Hello world!' }
```